### PR TITLE
Added new relics, ranger pet and icons

### PR DESCRIPTION
**Pull request by Linkaaaaa**
Added:

- Relic of the Twin Generals
- Relic of the Founding
- Juvenile Spinegazer

I have added icons for the new pet skills and merged soulbeast skills because the API doesn't have them.
The Spinegazer image will need an update once the correct pet art is datamined and uploaded to the wiki.

[logs.zip](https://github.com/baaron4/GW2-Elite-Insights-Parser/files/15400947/logs.zip)

### CODE REVIEW

File: GW2EIEvtcParser/ParsedData/Skills/SkillItem.cs [hunk 1]
```diff
@@ -760,6 +763,11 @@ internal static (long, long) GetArcDPSCustomIDs(int evtcVersion)
             { TakedownSmokescale, "https://wiki.guildwars2.com/images/e/e6/Takedown.png" },
             { PhasePounceWhiteTiger, "https://wiki.guildwars2.com/images/2/20/Phase_Pounce.png" },
             { PhotosynthesizeJacaranda, "https://wiki.guildwars2.com/images/2/2f/Photosynthesize.png" },
+            { EvilEye, "https://wiki.guildwars2.com/images/2/27/Evil_Eye.png" },
+            { TormentingVisionSpinegazer, "https://wiki.guildwars2.com/images/0/0b/Tormenting_Visions.png" },
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/ParsedData/Skills/SkillItem.cs:767]: Are those skill icons actually missing or is it because you were using an outdated API cache?

Comment (Linkaaaaa) [reply GW2EIEvtcParser/ParsedData/Skills/SkillItem.cs:767]: These don't exist
https://api.guildwars2.com/v2/skills?ids=72603&lang=en
https://api.guildwars2.com/v2/skills?ids=72527&lang=en
The other 3 do, I've added all for completition, should I remove them?
https://api.guildwars2.com/v2/skills?ids=72636&lang=en
https://api.guildwars2.com/v2/skills?ids=72843&lang=en
https://api.guildwars2.com/v2/skills?ids=72851&lang=en

Comment (EliphasNUIT) [reply GW2EIEvtcParser/ParsedData/Skills/SkillItem.cs:767]: If they exist in the API, yes, I'll update the API caches so they should appear naturally.

**Comment by EliphasNUIT** [comment]
Thanks!

**Pull request merged**